### PR TITLE
Add and show property features

### PR DESCRIPTION
**Pull request by Aakashbist**

### CODE REVIEW

File: src/components/services/imageUploadService.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Component } from 'react';
+import Firebase from '../../config/Firebase'
+
+
+export async function getdownloadImageUrl(uri, fileName) {
+    const imageUri = Platform.OS === 'ios' ? uri.replace('file://', '') : uri
+    const response = await fetch(imageUri);
+    const blob = await response.blob();
+    var storageRef = Firebase.storage().ref().child('images/' + fileName);
```
[the remark this answers is not in the document]
Comment (dilroop) [reply src/components/services/imageUploadService.js]: reading this function , it does more than it say. 

its uploading the file and getting the download link.

so name should take uploading aspect

File: src/components/useForms.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from 'react'
```
Comment (dilroop) [inline src/components/useForms.js]: seems like you are not using this file

File: src/components/validation.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+export default function validation(values) {
+    let errors = {};
```
Comment (dilroop) [inline src/components/validation.js]: seems like not using this file as well

File: src/resources/appRoute.js [hunk 4]
```diff
@@ -4,9 +4,11 @@ const AppRoute = {
     Auth: 'Auth',
     AuthLoading: 'AuthLoading',
     Dashboard: 'DashboardScreen',
-    Profile: 'ProfileScreen',
```
Comment (dilroop) [inline src/resources/appRoute.js]: was this being used some where?

File: src/resources/colors.js [hunk 5]
```diff
@@ -13,7 +13,10 @@ export default {
     black: "#000000",
     green: '#4CAF50',
     blue: '#3D6DCC',
+    red: '#ff0050',
 
     textColorError: "#b1003c",
     textColorSuccess: '#4CAF50',
+
+
```
Comment (dilroop) [inline src/resources/colors.js]: no need for blank lines in end

File: src/resources/styles.js [hunk 6]
```diff
@@ -49,11 +109,43 @@ export default {
...
+        height: 60,
+        marginTop: 0,
+        marginBottom: 0,
+        padding: 5,
+        backgroundColor: colors.darkWhite1,
+        borderBottomWidth: .25
+    },
+    map: {
+        height: 150
+
```
Comment (dilroop) [inline src/resources/styles.js]: remove blank lines like this

File: src/resources/appRoute.js (hunk 4 above)
Comment (Aakashbist) [reply src/resources/appRoute.js]: for later tenant and owner profile

File: src/components/validation.js (hunk 3 above)
Comment (Aakashbist) [reply src/components/validation.js]: leave it for later reusable class for validation

**Comment by Aakashbist** [comment]
property add module completed

**Pull request merged**